suite-native/app/package.json

```diff
@@ -23,6 +23,7 @@
     "dependencies": {
         "@gorhom/bottom-sheet": "5.0.5",
         "@mobily/ts-belt": "^3.13.1",
+        "@react-native-async-storage/async-storage": "^2.1.1",
```

Comment:
We use `react-native-mmkv` for perisistent storage, also because we can encrypt that.

Reply:
Unfortunately WalletConnect uses it directly, so I don't think I can get rid of it unless I made some of shim package. 
https://github.com/WalletConnect/walletconnect-utils/blob/master/misc/keyvaluestorage/src/react-native/index.ts

Reply:
Seems depcheck doesn't like that, should I try to work around it or move it back to app?